src/lib/ui/thermostat/ThermostatCard.tsx

```diff
@@ -0,0 +1,171 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+import type { ClimateSetting } from 'seamapi'
+
+import { FanIcon } from 'lib/icons/Fan.js'
+import { OffIcon } from 'lib/icons/Off.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+import { useDevice } from '../../../hooks.js'
+
+interface ThermostatCardProps {
+  deviceId: string
+}
+
+export function ThermostatCard({ deviceId }: ThermostatCardProps): JSX.Element {
+  return (
+    <div className='seam-thermostat-card'>
+      <Content deviceId={deviceId} />
+    </div>
+  )
+}
+
+function Content(props: { deviceId: string }): JSX.Element | null {
+  const { deviceId } = props
+
+  const { device } = useDevice({
```

Comment:
I'm not sure ThermostatCard will own the loading state vs the parent component. I believe the things inside `ui` are presentational components that assume data loading is handled by the parent.  Maybe this should take a `device` prop instead of a device_id?

Reply:
Sounds good to me. The reason I added `deviceId` initially was so I could easily use it in a story. How can I access specific devices from the seed-fake in a story?

Reply:
What if we just move the hook into the story?

Reply:
Oh duh lol—I keep thinking stories are on a different plane or something. Fixed, thanks!